beacon/pow/src/main/java/tech/pegasys/teku/beacon/pow/Eth1DepositManager.java

```diff
@@ -103,6 +108,23 @@ public void start() {
         .ifExceptionGetsHereRaiseABug();
   }
 
+  private SafeFuture<LoadDepositSnapshotResult> loadDepositSnapshot() {
+    return eth1SnapshotLoaderChannel
```

Comment:
I think I missed this in an earlier PR, but this isn't actually being used as an event channel (ie it's not going through the eventChannels.getPublisher() call on the calling side and a separate service providing the subscriber - it's just being given a direct implementation).  So we should simplify by removing the interface and just referring to `DepositSnapshotLoader` directly. Then it doesn't need to return a `SafeFuture` either since it's actually loading synchronously.

We generally only use EventChannels when communicating between separate services or major components. Internal to components like this we can just use normal calls.